packages/iov-keycontrol/src/keyring-entries/slip10.ts

```diff
@@ -261,4 +280,29 @@ export class Slip10KeyringEntry implements KeyringEntry {
     const keypair = await Ed25519.makeKeypair(derivationResult.privkey);
     return keypair;
   }
+
+  private buildLocalIdentity(bytes: PublicKeyBytes, label: string | undefined): LocalIdentity {
+    const algorithm = Slip10KeyringEntry.algorithmFromCurve(this.curve);
+    const pubkey: PublicKeyBundle = {
+      algo: algorithm,
+      data: bytes,
+    };
+    return {
+      pubkey,
+      label,
+      id: Slip10KeyringEntry.identityId({ pubkey }),
+    };
+  }
+
+  // calculate id returns the tripple sha256 hash of the bip39 entropy as hex-string
```

Comment:
secret has enough entropy to be brute-force resistant. So the triple hash is not necessary (and would not help). But it would be nice to have a use case identifier in the hash to avoid any kind of collisions with results in other applications. Using HMAC-SHA256 we can add something like "IOV entry ID" to the secret.

Reply:
sounds good.